src/shogun/classifier/svm/LibLinear.cpp

```diff
@@ -67,24 +67,23 @@ CLibLinear::~CLibLinear()
 {
 }
 
-bool CLibLinear::train_machine(CFeatures* data)
+bool CLibLinear::train_machine(CFeatures* features)
 {
+	ASSERT(m_labels);
+	train_machine(features, m_labels);
+	return true;
+}
 
-	ASSERT(m_labels)
-	init_linear_term();
+void CLibLinear::train_machine(CFeatures* features, CLabels* labels)
```

Comment:
const possible?

Reply:
There are many non-const methods algorithm internal methods. We can either use non-const arguments here, or use `const_cast` to drop const in internal methods

Reply:
@karlnapf One problem ref counting is `IterativeMachine`. We need to increase ref count of features and labels in `init_model`

Reply:
ah crap yes, the ref counting is non-const....
ok so we need to postpone the const making for now until we have another way to do the reference counting